Draft improvements from assistant conversations: run moves and nested config (#2830)
wandb/docs: models/track/config.mdx

python train.py --configs other-config.yaml
```

### Nested dictionaries in config

> The `desc`/`value` schema used by `config-defaults.yaml` does not support nested dictionaries inline. To log a nested dictionary as config, pass it directly to `wandb.init()` in Python:

Reviewer: ❌ **Refuted** — the source says this is wrong.

```suggestion
`config-defaults.yaml` supports nested dictionaries when the nested mapping is the top-level key's `value`. You can also pass a nested dictionary directly to `wandb.init()` in Python:
```

**Source says:** the loader extracts each top-level `value` without constraining its type, and the backend explicitly flattens map values. · **Sources:** [wandb:config_util.py#L66](https://github.com/wandb/wandb/blob/a2cb871814a97cb7094fb34f014c3cf5d1287c34/wandb/sdk/lib/config_util.py#L66-L95); [core:utils.go#L389](https://github.com/wandb/core/blob/df3198f3c93a9da1e6c3313d3f406140d70575d3/services/gorilla/metadata/experiments/utils.go#L389-L446)

_🤖 CoreWeave Docs Team's Validator — refuted_
<!-- cw-docs-validator -->
